src/main/java/uk/gov/justice/probation/courtcasematcher/restclient/OffenderSearchRestClient.java

```diff
@@ -131,26 +96,23 @@ private MatchRequest buildRequestBody(String pnc, Name fullName, LocalDate dateO
     /**
      * Filter which decides whether or not to retry. Return true if we do wish to retry.
      */
-    static final Predicate<? super Throwable> EXCEPTION_RETRY_FILTER = new Predicate<Throwable>() {
-        @Override
-        public boolean test(Throwable throwable) {
-            boolean retry = true;
-            if (throwable instanceof WebClientResponseException) {
-                WebClientResponseException ex = (WebClientResponseException) throwable;
-                HttpStatus status = ex.getStatusCode();
-                switch (status) {
-                    case NOT_FOUND:
-                    case FORBIDDEN:
-                    case UNAUTHORIZED:
-                    case TOO_MANY_REQUESTS:
-                        retry = false;
-                        break;
-                    default:
-                        retry = true;
-                }
+    static final Predicate<? super Throwable> EXCEPTION_RETRY_FILTER = throwable -> {
```

Comment:
This is a style issue . I saw that IntelliJ suggested it but didn't think using a lambda syntax really helped readability.